home/src/main/resources/rdf/i18n/en_US/tbox/firsttime/searchOntology.n3

```diff
@@ -0,0 +1,32 @@
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
```

Comment:
Should we call this file searchLabels? How those labels are used? Do we need its i18n or it is used only in logs?

Reply:
This file contains labels only related to search ontology, not translations used on search results pages.

Reply:
renamed